app/src/main/java/com/parking_system_kotlin/utils/Constants.kt

```diff
@@ -0,0 +1,6 @@
+package com.parking_system_kotlin.utils
+
+object Constants {
+    var LISTENER_SPACES_SETTING_KEY: String = "LISTENER_SPACES_SETTING_KEY"
```

Comment:
these constants should be **const val**, not var if you set it like this yous are setting a mutable variable

Reply:
done